script.js

```diff
@@ -103,6 +102,19 @@ function searchFourSquare(search) {
   }); // Completes the function that pulls down the response
 } // Completes the entire function that searches foursquare
 
+// Initialize search
+function initSearch() {
+  searchBar = true;
```

Comment:
Is this still necessary? Trying to find where else this is used and not seeing it now

Reply:
I don't see what this comment is referring too